### PR TITLE
Show message when no organization is assigned to the user

### PR DESCRIPTION
**Pull request by Jayendra88**
#### What does this PR do? 
This PR is to display a message when user is not assigned to any organization

#### How to test it?

go to the my account page and click on the organizations
[https://clouda--usdealerdev.myvtex.com/](https://clouda--usdealerdev.myvtex.com/)

#### Screen shot
image

### CODE REVIEW

**Comment by Jayendra88** [comment]
> Hi! I'm VTEX IO CI/CD Bot and I'll be helping you to publish your app! 🤖
> 
> Please select which version do you want to release:
> 
> * [x ]  Patch (backwards-compatible bug fixes)
> * [ ]  Minor (backwards-compatible functionality)
> * [ ]  Major (incompatible API changes)
> 
> And then you just need to merge your PR when you are ready! There is **no need** to create a release commit/tag.
> 
> * [ ]  No thanks, I would rather do it manually 😞

**Comment by Jayendra88** [comment]
> Beep boop 🤖
> 
> I noticed you didn't make any changes at the `docs/` folder
> 
> * [x ]  There's nothing new to document 🤔
> * [ ]  I'll do it later 😞
> 
> In order to keep track, I'll create an issue if you decide now is not a good time
> 
> * [ ]  I just updated 🎉🎉

File: messages/context.json [hunk 1]
```diff
@@ -69,5 +69,7 @@
   "store/my-users.my-organization.status.approved": "Approved",
   "store/my-users.my-organization.status.not-approved": "Not Approved",
   "store/my-users.my-organization.status.isOrgAdmin": "Organization Admin",
-  "store/my-users.my-organization.showMore": "Show More"
-}
\ No newline at end of file
+  "store/my-users.my-organization.showMore": "Show More",
+  "store/my-users.my-organization.not-yet-assigned-1": "You are not yet assigned to any organization. ",
```
Comment (lucis) [inline messages/context.json:73]:
```suggestion
  "store/my-users.my-organization.not-yet-assigned-1": "You are not yet assigned to any organization.",
```

File: messages/en.json [hunk 2]
```diff
@@ -69,5 +69,7 @@
   "store/my-users.my-organization.status.approved": "Approved",
   "store/my-users.my-organization.status.not-approved": "Not Approved",
   "store/my-users.my-organization.status.isOrgAdmin": "Organization Admin",
-  "store/my-users.my-organization.showMore": "Show More"
+  "store/my-users.my-organization.showMore": "Show More",
+  "store/my-users.my-organization.not-yet-assigned-1": "You are not yet assigned to any organization. ",
```
Comment (lucis) [inline messages/en.json:73]:
```suggestion
  "store/my-users.my-organization.not-yet-assigned-1": "You are not yet assigned to any organization.",
```

**Review by lucis — APPROVED** [review]

**Review by gsasouza — APPROVED** [review]
LGTM

**Pull request merged**